src/app/home/home.page.ts

```diff
@@ -21,6 +21,9 @@ register();
   styleUrls: ['./home.page.scss'],
 })
 export class HomePage {
+  spotlightstories:any[]=[];
+  myImprovements: any[] = [];
+  recommendationList:any[]=[];
   formListingUrl = (environment.baseURL.includes('project') ?  urlConfig.subProject : urlConfig.subSurvey ) + urlConfig['formListing'].listingUrl;
```

Comment:
can we remove this code block if not needed

Reply:
to work the carousels we need the arrays and it is been using in the cards might we need different alternative cards to replace if there is no data

Reply:
meant this logic formListingUrl @farhanp1502

Reply:
removed the unwanted the block